shared/actions/settings.tsx

```diff
@@ -393,6 +395,21 @@ const loadSettings = () =>
     })
   })
 
+const computeBadgesOnLoadedSettings = (state, action: SettingsGen.LoadedSettingsPayload) => {
+  const emailCount = (action.payload.emails || I.Map())
+    .map(emailRow => (emailRow.isVerified ? 0 : 1))
```

Comment:
I think this would be cleaner as a filter-reduce instead of map-reduce.

Reply:
why not just do reduce, you don't need a map or filter first

Reply:
Hah yes, good call.